clang/test/CheckedCRewriter/b12_callerstructnp.c

```diff
@@ -8,48 +10,61 @@ extern _Itype_for_any(T) void *realloc(void *pointer : itype(_Array_ptr<T>) byte
 extern int printf(const char * restrict format : itype(restrict _Nt_array_ptr<const char>), ...);
 extern _Unchecked char *strcpy(char * restrict dest, const char * restrict src : itype(restrict _Nt_array_ptr<const char>));
 
+
 struct np {
-  int x;
-  int y;
+    int x;
+    int y;
 };
 
 struct p {
-  int *x;
-  char *y;
+    int *x;
+	//CHECK: int *x;
+    char *y;
+	//CHECK: char *y;
 };
 
+
 struct r {
-  int data;
-  struct r *next;
+    int data;
+    struct r *next;
+	//CHECK: _Ptr<struct r> next;
 };
 
+
 struct np *sus(struct p x, struct p y) {
+	//CHECK_NOALL: struct np *sus(struct p x, struct p y) : itype(_Ptr<struct np>) {
+	//CHECK_ALL: struct np * sus(struct p x, struct p y) {
```

Comment:
This is very strange: Enabling `-alltypes` results in a WILD pointer whereas not enabling it results in a checked pointer. I don't know why that would happen. We can accept this PR, but perhaps put this on your list to investigate?

Reply:
Might have something to do with how `malloc` is handled in the two cases.

Reply:
This is weird. Looking into it now